andela/firestar-backend
# #167730727 Setup Postgres

#### What does this PR do?
Configure BareFoot Nomad to use PostgreSQL as DB Engine
#### Description of Task to be completed?
* Setup Sequelize
#### How should this be manually tested?
* git clone this branch [ch-setup-postgres-167730728](https://github.com/andela/firestar-backend/tree/ch-setup-postgres-167730727)
* cd to the project
* Checkout to the branch
* Check `package.json` file to see `pg` dependency

#### Any background context you want to provide?
* N/A

#### What are the relevant pivotal tracker stories?
[#167730727](https://www.pivotaltracker.com/story/show/167730727)

#### Screenshots or gifs (if appropriate)
* N/A
#### Questions:
* N/A
